install.sh

```diff
@@ -197,7 +197,8 @@ setup_env() {
     if [ -n "${INSTALL_K3S_SYSTEMD_DIR}" ]; then
         SYSTEMD_DIR="${INSTALL_K3S_SYSTEMD_DIR}"
     else
-        SYSTEMD_DIR=/etc/systemd/system
+        mkdir -p /usr/local/etc/systemd/system
```

Comment:
Should we change this to be `mkdir -p ${SYSTEMD_DIR}` after this `if` block?

Reply:
It's a while since I was dealing with this but reordering might look more logical. I'm not sure you'll want to unconditionally try to create the directory since /etc/systemd/system/ should always exist on a systemd init OS.

Reply:
There seems to be 2 problems here:
First, a unit file created by system administrator can, and by default it’s assumed to, be stored into /etc/systemd/system directory (not a symlink of it, as the PR states).
Second, the recommended location for software related unit files, installed by an administrator, which could be considered the case here, is /user/local/lib/systemd/system, which is part of system unit path, in both system and user modes.
The proposed path of /user/local/etc/systemd/system is not part of the unit path of system or user modes; so, if this ever gets merged, then, “systemctl link” command, or setting SYSTEMD_UNIT_PATH environment variable will be required as well.